SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -441,6 +443,7 @@ class PositioningViewController: UIViewController, GMSMapViewDelegate, UITableVi
     }
     
     func select(floor floorIndex: IndexPath) {
+        self.stairs.map = nil
```

Comment:
Non habería que facelo despois de comprobar efectivamente se houbo un cambio de piso?

Reply:
Podes crear unha funcion cun nome que explique o que se está facendo aí, que basicamente e resetar os markers de cambio de planta. Así o código queda moito máis lexible